kpatch-build/create-diff-object.c

```diff
@@ -917,87 +917,95 @@ do {								\
 	elem->twin = NULL;					\
 } while (0)
 
-static void __kpatch_correlate_section(struct section *sec1, struct section *sec2)
+static void __kpatch_correlate_section(struct section *sec_base,
+		struct section *sec_patched)
 {
-	CORRELATE_ELEMENT(sec1, sec2, "section");
+	CORRELATE_ELEMENT(sec_base, sec_patched, "section");
```

Comment:
This is a nice cleanup IMO.  The new naming helps to codify certain existing assumptions.

While making such a broad change, I'd suggest one more naming tweak (but arguments to the contrary are welcome).  I've long regretted the use of the term "base" in this code.  I'd argue that "orig" is more descriptive than "base" (and matches the directory names in ~/.kpatch/tmp/.)  Since you're already doing a mass renaming which adds more "base", I'd suggest going ahead and doing s/base/orig/g pretty much everywhere -- if there are no objections.

And sorry about pulling this PR into the weeds :-)